schemas/context/identity.schema.json

```diff
@@ -0,0 +1,38 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/identity",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Identity",
+  "type": "object",
+  "description": "Identity from identity services which is the fact of being who or what a person or thing is.\n",
```

Comment:
This description does not make any sense. The `identity` here is neither describing who or what a person is (does not cover things at all, or only tangentially), but a unique value that has been associated with a person, device, or user agent by an identity service.

If we use the name "identity service" we need to explain how it is related to the namespace in this schema.

Reply:
Agreed, we should make this more clear